packages/base/src/asset-registries/Icons.ts

```diff
@@ -138,6 +139,37 @@ const getIconData = async (name: string) => {
 	return registry.get(registryKey);
 };
 
+/**
+ * Returns the accessible name for the given icon,
+ * or undefined if accessible name is not present.
+ *
+ * @param { string } name
+ * @return { Promise }
+ */
+const getIconAccessibleName = async (name: string) => {
+	let iconData: typeof ICON_NOT_FOUND | IconData | undefined = getIconDataSync(name);
+	if (!iconData) {
+		iconData = await getIconData(name);
+	}
+
+	if (!iconData) {
+		/* eslint-disable-next-line */
+		console.warn(`Required icon is not registered. Invalid icon name: ${name}`);
```

Comment:
can we keep it simpler and not log any warning here? this method will always be used with an actual icon, and it will display the warnings anyway.

if we remove the logs here, we will skip the duplication in the console and there will be less code. what do you think?

Reply:
less code, looks better